### ISSUE TITLE
Error Starting KenyaEMR charts Module

### ISSUE THREAD

**Issue by karimkkanji** [karimkkanji/kenyaemr-plus-issues#3]
The Kenyaemr_etl and kenyaemr_datatools have been dropped and recreated using the following:

call create_etl_tables();
call sp_first_time_setup();
 
In mysql console and Tomcat restarted but the issue persists.

The following is an error observed in the console:

![image](https://user-images.githubusercontent.com/18611345/126548955-5529522b-3174-4dac-882e-b929ede524cf.png)

**Comment by ngarivictor** [comment]
you could try source a fresh 👇
[kenyaemr_etl.zip](https://github.com/karimkkanji/kenyaemr-plus-issues/files/6859912/kenyaemr_etl.zip)

**Comment by karimkkanji** [comment]
Let me try again. Have tried deleting the databases and recreating them but the issue still persists

**Comment by karimkkanji** [comment]
The ETL Database shared has actually made it worse. Home Page graphs are now not appearing at all

**Comment by karimkkanji** [comment]
Running the following on MYSQL Shell might do the trick:

```sql
DROP DATABASE kenyaemr_etl;
DROP DATABASE kenyaemr_datatools;
USE openmrs;
CALL create_etl_tables();
CALL sp_first_time_setup();
```

**Comment by ngarivictor** [comment]
Let me know how it goes.

**Comment by karimkkanji** [comment]
@ngarivictor It is a database issue. I have used the current database running in at the facility and it now works well. All modules work well. I had tried with the de-id database and the etl database you shared but that didn't work.

Issue: The openmrs Database might be having a schema issue or LIQUIBASE issue
